config/default.yaml

```diff
@@ -80,6 +80,19 @@ executor:
                     high: 12
         # Container tags to use
         launchVersion: stable
+    nomad:
```

Comment:
Can you comment this part out just like jenkins? Our team is not supporting this by default. If someone wants to use it, can use the custom-environment-variables.yaml.